tests/vectors_sig.c

```diff
@@ -255,6 +332,11 @@ static int sig_ver_vector(const char *method_name,
 	} else if (!strcmp(method_name, "ML-DSA-87")) {
 #ifdef OQS_ENABLE_SIG_ml_dsa_87
 		rc = pqcrystals_ml_dsa_87_ref_verify_internal(sigVer_sig_bytes, sig->length_signature, sigVer_msg_bytes, msgLen, NULL, 0, sigVer_pk_bytes);
+#endif
+	} else if (!strncmp(method_name, "SLH_DSA", 7)) {
```

Comment:
I am not sure about the logic of the macro here.
If `OQS_ENABLE_SIG_SLH_DSA` is turned off, then having `method_name` set to `"SLH_DSA_XXX"` should be considered invalid, so the routine should `goto err`.
Instead, this block says if the macro is turned off, and `method_name` is set to `"SLH_DSA_XXX"`, then do nothing. Is this the intended behavior?